tests/other_test.go

```diff
@@ -18,7 +18,7 @@ func TestRunRestoredDBExample(t *testing.T) {
 		BestRegionYAMLPath: regionSelectionPath,
 		ResourceGroup:      resourceGroup,
 		TerraformVars: map[string]interface{}{
-			"pg_version": "13",
+			"pg_version": "14",
```

Comment:
The PITR test is testing version 14, so set this back to 15 and we should then have 1 test for every version

Reply:
Changed that to 13 again, as:

FSCloud: 15
PITR: 14
RestoreDB: 13
Basic example: 12
Upgrade test: 11